### ISSUE TITLE
Option for not having pronunciations? Japanese/Chinese

### ISSUE THREAD

**Issue by ghost** [migaku-official/Migaku-Dictionary-Addon#56]
**Type:**
  - [ ] bug
  - [ x] feature
  - [ ] enhancement
  - [ ] question

**Environment:**
  - OS: Linux x64

**I'm going to open a PR to resolve the issue:**
  - [ ] yes
  - [ x] no

**Description:**
Does the possibility exist to not have the bracketed/alternative pronunciations before the definitions of a word? I mean these:
【簗】 やな 
やな【梁・簗】
--Definition--

I use tts to play the definition and it's pretty cumbersome to hear all the alternative sounds. I can remove everything within brackets with awesometts but there's still the other part.

**Comment by soliviantar** [comment]
Agreed. I tried creating definitions from the bilingual dictionary for the front of recall cards, but since it includes both the hanzi and the pinyin, I can't really use it for that.

It's ok for the definition field where I put the monolingual definition since I am not using those for the front at this point, but still, I think the headwords are not necessary since they are already in another field.

Thanks for the addon in anycase!

**Comment by KieranBrannigan** [comment]
Thank you for supporting Migaku.
We apologise that recently we didn't have the manpower to maintain these repos properly, but we aim to maintain them better from here on out.

I think you can achieve what you're asking by editing the Term Header for that particular dictionary. You can remove the "pronunciation" part, then the pronunciation won't be exported. Hope this helps.

![image](https://user-images.githubusercontent.com/60760551/114465773-7a7da900-9bdf-11eb-9428-a33e780436d3.png)

Unfortunately, the text within the definition itself can't be easily edited when exporting.

Please let me know if this solution works so we can close the issue. 

Happy immersing!
Kieran
